style.css

```diff
@@ -4,13 +4,54 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+  scroll-behavior: smooth;
 }
 
 body {
   background-color: #e5e5e5;
   font-family: 'Poppins', sans-serif;
 }
 
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  filter: brightness(0.8);
+}
+
+a:visited {
+  color: none;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+  opacity: 0.9;
+  box-shadow: 0 0 16px 0 rgba(64, 83, 252, 0.24);
```

Comment:
- [ ] I would like to recommend you remove the `box-shadow` in making your navigation similar to the one on the Figma template . Please fix it :+1: